front/app/containers/Admin/pages/index.tsx

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export default ({ children }) => <>{children}</>;
```

Comment:
Not sure about the reason for his component, but I suppose you have just used the old code? Also, it seems to work so it's fine I guess.

Reply:
It's just a placeholder needed for the router configuration, used in a few other places as well